frontstage/models.py

```diff
@@ -14,43 +14,45 @@
     first_name = StringField('First name',
-                             validators=[InputRequired("First name is required"),
+                             validators=[InputRequired('First name is required'),
                                          Length(max=254,
                                                 message='Your first name must be less than 254 characters')])
     last_name = StringField('Last name',
-                            validators=[InputRequired("Last name is required"),
+                            validators=[InputRequired('Last name is required'),
                                         Length(max=254, message='Your last name must be less than 254 characters')])
     email_address = StringField('Enter your email address',
-                                validators=[InputRequired("Email address is required"),
+                                validators=[InputRequired('Email address is required'),
                                             Email(message="Your email should be of the form 'myname@email.com' "),
                                             Length(max=254,
                                                    message='Your email must be less than 254 characters')])
     password = PasswordField('Create a password',
-                             validators=[DataRequired("Password is required"),
+                             validators=[DataRequired('Password is required'),
                                          EqualTo('password_confirm', message=app.config['PASSWORD_MATCH_ERROR_TEXT']),
                                          Length(min=app.config['PASSWORD_MIN_LENGTH'],
                                                 max=app.config['PASSWORD_MAX_LENGTH'],
                                                 message=app.config['PASSWORD_CRITERIA_ERROR_TEXT'])])
     password_confirm = PasswordField('Re-type your password')
     phone_number = StringField('Enter your phone number',
-                               validators=[DataRequired("Phone number is required"),
+                               validators=[DataRequired('Phone number is required'),
                                            Length(min=9,
                                                   max=15,
-                                                  message="This should be a valid phone number between 9 and 15 digits")],
+                                                  message='This should be a valid phone number between 9 and 15 digits')],
                                default=None)
     enrolment_code = HiddenField('Enrolment Code')
 
     @staticmethod
     def validate_phone_number(form, field):
         try:
             logger.debug('Checking this is a valid GB phone number')
-            input_number = phonenumbers.parse(field.data, "GB")  # Tell the parser we are looking for a GB number
+            input_number = phonenumbers.parse(field.data, 'GB')  # Tell the parser we are looking for a GB number
```

Comment:
Is there a reason why we're only accepting GB numbers when a lot of respondents are overseas?

Reply:
I'm not sure, theres a car for this work, so probably look to add it on there:

Reply:
Ok looking like we are going to remove this added validation, so just to ignore this @gavinleeedwards

Reply:
All good if these bits are captured on other tickets :+1: